hack/verify-description.sh

```diff
@@ -44,9 +48,9 @@ else
 fi
 
 for file in $versioned_api_files; do
-  if grep json: "${file}" | grep -v // | grep -v ,inline | grep -v -q description: ; then
-    echo "API file is missing the required field descriptions: ${file}"
-    result=1
+  $genswaggertypedocs -v -s "${file}" -f - || result=$?
+  if [[ "${result}" -ne "0" ]]; then
+    echo "API file: ${file} is missing: ${result} descriptions"
```

Comment:
We should extend the check below to ensure that the versioned API files also do not contain description tags.